enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -0,0 +1,50 @@
+(ns metabase-enterprise.audit-db
+  "This is here so we can try to require it and see whether or not EE code is on the classpath."
+  (:require [metabase.db.env :as mdb.env]
+            [metabase.models.database :refer [Database]]
+            [metabase.util :as u]
+            [metabase.util.log :as log]
+            [toucan2.core :as t2]))
+
+(def ^:private default-admin-db-id 13371337)
+
+(defn- install-database!
+  "Creates the audit db, a clone of the app db used for auditing purposes.
+
+  - This uses a weird ID because some tests are hardcoded to look for database with ID = 2, and inserting an extra db
```

Comment:
Not sure how many, but here are examples:

1. [setting this ID, and using it to look up postgres](https://github.com/metabase/metabase/blob/bcm-add-app-db-as-audit-db/e2e/test/scenarios/models/model-actions.cy.spec.js#L25)
2. [setting this ID and using it to look up mysql](https://github.com/metabase/metabase/blob/bcm-add-app-db-as-audit-db/e2e/test/scenarios/admin/datamodel/editor.cy.spec.js#L20)

So, the tests issue a request to api/database/{id}, which are sequential, hence will be different if a new db was inserted into metabase_database. 😓

Reply:
Ohh I see, I thought you meant backend tests.

cc @nemanjaglumac — do you think there's any way for Cypress to get the IDs of these test databases more directly, rather than hardcoding or assuming it's one above the sample DB?

Don't think this needs to block this PR though